proposals/function-pointers.md

```diff
@@ -63,174 +64,173 @@ unsafe class Example {
 delegate Func1 Func2(Func1 f);
 
-// Function pointer equivalent without parens or calling convention
-func* int(string);
-func* func* int(string) int(func* int(string));
-
-// Function pointer equivalent without parens and with calling convention
-func* managed int(string);
-func* managed func* managed int(string) int(func* managed int(string));
-
-// Function pointer equivalent with parens and without calling convention
-func* int(string);
-func* (func* int(string)) int((func* int(string));
-
-// Function pointer equivalent of with parens and calling convention
-func* int(string)
-func* managed (func* managed int(string)) int((func* managed int(string));
-```
-
-When the calling convention is omitted from the syntax then `managed` will be used as the calling convention. That means
-all of the forms of `Func1` and `Func2` defined above are equivalent signatures.
+// Function pointer equivalent without calling convention
+delegate*<string, int>;
+delegate*<delegate*<string, int>, delegate*<string, int>>;
 
-The calling convention cannot be omitted when the return type of the function pointer has the same name as a calling 
-convention. In that case, the parser would process the return type as a calling convention instead of a type. To resolve
-this the developer must specify both the calling convention and the return type. 
-
-``` csharp
-class cdecl { }
-
-// Function pointer which has a cdecl calling convention, a cdecl return type and takes a single 
-// parameter of type cdecl;
-func* cdecl cdecl(cdecl);
+// Function pointer equivalent with calling convention
+delegate* managed<string, int>;
+delegate*<delegate* managed<string, int>, delegate*<string, int>>;
 ```
 
 ### Allow address-of to target methods
 
-Method groups will now be allowed as arguments to an address-of expression. The type of such an 
-expression will be a `func*` which has the equivalent signature of the target method and a managed 
+Method groups will now be allowed as arguments to an address-of expression. The type of such an
+expression will be a `delegate*` which has the equivalent signature of the target method and a managed
 calling convention:
 
 ``` csharp
-unsafe class Util { 
-    public static void Log() { } 
+unsafe class Util {
+    public static void Log() { }
 
     void Use() {
-        func* void() ptr1 = &Util.Log;
+        delegate*<void> ptr1 = &Util.Log;
 
-        // Error: type "func* void()" not compatible with "func int()";
-        func* int() ptr2 = &Util.Log; 
+        // Error: type "delegate*<void>" not compatible with "delegate*<int>";
+        delegate*<int> ptr2 = &Util.Log;
 
         // Okay. Conversion to void* is always allowed.
         void* v = &Util.Log;
    }
 }
 ```
 
-The conversion of an address-of method group to `func*` has roughly the same process as method group to `delegate`  
+The conversion of an address-of method group to `delegate*` has roughly the same process as method group to `delegate`
```

Comment:
>address-of method group [](start = 21, length = 23)

I think you may need to introduce and define the concept of an "address-of method group" if you want to say things about it.